Library/BarPlotsMultipleVariables.Rmd

```diff
@@ -22,70 +24,76 @@ packages <- c(
 pacman::p_load(packages, character.only = TRUE, install = FALSE)
 
 # Load an example dataset ---------------
-data <- read_dta("https://github.com/worldbank/r-econ-visual-library/raw/master/Library/Data/BarPlotsMultipleVariables.dta")
+data <- readRDS("~/Documents/GitHub/DIME/Code_Reviews/CodeReview_r-econ-visual-library/r-econ-visual-library/Library/BarPlotsMultipleVariables.rds")
 
 # Printing the first five rows of the data to visualize its structure before data wrangling
 head(data)
 
+# Data Dictionary ---------------
+  ## Data Source: This graph uses fictional data created for the purpose of this graph. The fictional data comes from an imagined survey of tropical farmers participating in a fictional randomized control trial. 
+  ## Tidy format? Yes
+  ## Unit of Observation: Households
+  ## Variables of interest
+    # hh_id: Household identifier
+    # round: Survey round id (baseline or midline) 
+    # treatment_group: Treatment status id
+    # w_Papaya: Annual Papaya Production
+    # w_Pineapple: Annual Pineapple Production
+    # w_Coconut: Annual Coconut Production 
+
+
 # Data wrangling ---------------
```

Comment:
Explain why this data wrangling is needed (in the text): the data is collected at household level, but graph will show the average by treatment group and round of data collection, so we need to get the data into this level first.